### DISCUSSION TITLE
Deploy to Dokploy

### DISCUSSION THREAD

**Discussion by niklasenB** [langgenius/dify#4378] [General]
I am trying to deploy dify using dokploy, running into an error on /install

where it fails on
http://127.0.0.1:5001/console/api/setup

the dokploy instance is running on a hetzner server using the docker image from docker hub - has anyone got experience how to fill / add the environment vars for this onprem installation?
![image](https://github.com/langgenius/dify/assets/13793903/1cf5c52f-2c85-429a-8642-39c114595b29)

**Comment by omesh845** [comment]
any reverse proxy selfhosted instances failed as per my experience, coolify and even tried individual caddy,traefik,nginx everything ends up in the same position as '/install'

**Comment by ravibaskaran** [comment]
Has anyone looked into the issue of Docker installation on dokploy? Did anyone succeed?

**Comment by jr-k** [comment]
Dokploy starts "docker compose up" from root directory and not docker directory, that's the problem...
